<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Page2</class>
 <widget class="QWizardPage" name="Page2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Paso 2: Malla y Parámetros de Simulación</string>
  </property>
  <property name="subTitle">
   <string>Cargue la malla y configure los parámetros básicos de la simulación.</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="meshGroupBox">
     <property name="title">
      <string>Carga de Malla (.unv, blockMeshDict)</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="meshPathLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="browseMeshButton">
        <property name="text">
         <string>Examinar...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="controlDictGroupBox">
     <property name="title">
      <string>Parámetros de Simulación (controlDict)</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="solverLabel">
        <property name="text">
         <string>Solver:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="solverLineEdit">
        <property name="text">
         <string>interFoam</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="startTimeLabel">
        <property name="text">
         <string>Tiempo de Inicio (startTime):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="startTimeLineEdit">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="endTimeLabel">
        <property name="text">
         <string>Tiempo Final (endTime):</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="endTimeLineEdit">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="deltaTLabel">
        <property name="text">
         <string>Paso de Tiempo (deltaT):</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="deltaTLineEdit">
        <property name="text">
         <string>0.01</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="writeIntervalLabel">
        <property name="text">
         <string>Intervalo de Escritura (writeInterval):</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="writeIntervalLineEdit">
        <property name="text">
         <string>0.1</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
